test/test_utils.py

```diff
@@ -50,3 +54,107 @@ def test_repr_safe_str(value, repr_safe_value):
     res = repr_safe_str(value)
     assert res == repr_safe_value
     assert isinstance(res, bytes)
+
+
+class TestGetRedactedArgs(object):
+
+    @pytest.mark.parametrize("sensitive_variables, expected", [
+        (tuple(), {'a': 'A', 'b': 'B'}),  # no sensitive variables
```

Comment:
I find this more obvious -- easier to distinguish it amongst all the other parenthesis. Happy to change if you'd prefer though.